en/docs/deploy-and-publish/publish-on-dev-portal/third-party-api-support.md

```diff
@@ -7,19 +7,23 @@ In previous versions of API Manager, third-party APIs were available when import
 
 ## Creating a Third-party API
 
+#### Method 1 - APIs already supported by API Manager
+
 1. A regular API can be converted to a third-party API from the **Basic Info** section in the API Publisher portal.
 
-!!! note
-    An API cannot be converted to a third-party API when there are active deployments.
+    !!! note
+        An API cannot be converted to a third-party API when there are active deployments.
 
-[![Convert to third-party API]({{base_path}}/assets/img/develop/convert-to-third-party.png)]({{base_path}}/assets/img/develop/convert-to-third-party.png)
+    [![Convert to third-party API]({{base_path}}/assets/img/develop/convert-to-third-party.png)]({{base_path}}/assets/img/develop/convert-to-third-party.png)
 
-2. In addition to the streaming APIs supported in API Manager (WebSocket, SSE and WebSub), the **Other** option provided when importing AsyncAPI definitions can be used to import a third-party API.
+#### Method 2 - Streaming and Async APIs
```

Comment:
Shall we make this "Streaming APIs"? since both the terms refer to the same API types